### PR TITLE
Content page

### PR DESCRIPTION
**Pull request by samtimalsina**
Issue: #85 

This adds a basic content page at /admin/content, but only lists articles at the moment.

screen shot 2018-04-13 at 1 04 01 pm

### CODE REVIEW

File: src/utils/api/api.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ async function api(endpoint, init = {}) {
     case 'roles':
       url = '/jsonapi/user_role/user_role';
       break;
+    case 'content':
```
Comment (dawehner) [inline src/utils/api/api.js]: So yeah we miss the feature to fetch nodes from multiple bundles here :(

**Comment by justafish** [comment]
Thanks @samtimalsina!

This route will need adding to [admin_ui_support.routing.yml](https://github.com/jsdrupal/drupal-admin-ui/blob/master/admin_ui_support/admin_ui_support.routing.yml) so that the redirect from Drupal works

The linting tests also need fixing

This is great! 🎉

File: src/utils/api/api.js (hunk 1 above)
Comment (tedbow) [reply src/utils/api/api.js]: #96 would allow this!

File: src/utils/api/api.js [hunk 2]
```diff
@@ -8,6 +8,9 @@ async function api(endpoint, init = {}) {
...
+      url = '/jsonapi/node/article';
```
Comment (tedbow) [inline src/utils/api/api.js]: Now that https://github.com/jsdrupal/drupal-admin-ui/pull/96 was merged you should be able to change this to `/json/node` to query all node entities.

**Comment by mattgrill** [comment]
@samtimalsina I've closed this, and taken your work (co-authored a commit) and added upon that work to request all the content from all the bundles. It's all in #101

**Pull request closed without merging**